### PR TITLE
Refactor/merge apis

### PR DESCRIPTION
**Pull request by Redm4x**
Add all stats to the new api and refactor website to use the new api for everything. Will remove old api files in another PR.

### CODE REVIEW

File: newApi/src/providers/marketDataProvider.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
+const fetch = require("node-fetch");
+
+const reftreshInterval = 5 * 60 * 1000; // 5min
+
+let aktMarketData = null;
+
+exports.syncAtInterval = async () => {
```
Comment (baktun14) [inline newApi/src/providers/marketDataProvider.js:7]: export const

**Comment by baktun14** [comment]
- Change lease to deployments with lease for total
- Fix numbers to match current system

**Comment by Redm4x** [comment]
Refactor Dockerfile/deploy.yml

File: newApi/src/akash/akashSync.ts [hunk 2]
```diff
@@ -135,10 +135,17 @@ async function insertBlocks(startHeight, endHeight) {
...
+    include: [
+      {
+        model: Day,
+        required: true
+      }
+    ],
     order: [["height", "DESC"]]
-  });
-  let lastInsertedDate = lastInsertedBlock?.datetime ?? new Date(2000, 1, 1);
+  })) as any;
```
[the remark this answers is not in the document]
Comment (Redm4x) [reply newApi/src/akash/akashSync.ts:147]: Will refactor this part in a future PR

**Pull request merged**